### DISCUSSION TITLE
False positive results from my execution

### DISCUSSION THREAD

**Discussion by PanagiotisDrakatos** [gunnarmorling/1brc#639] [Rule clarifications]
@gunnarmorling  i have a small problem with my execution and I am wondering in my local PC, it needs only 5 sec to get executed except only the first time which needs 50 sec. Why in the leaderboard result of my implementation is approximately 24 sec??  Here are my specs:

```
Intel core i9 12900K
Memory DDR5 64GB
Gigabyte Technology Co. Ltd. Z690 AORUS PRO (U3E1
```)

**Comment by gunnarmorling** [comment]
Hum, interesting. Do you have any idea why the first run is taking so much
longer? Note that as per the rules, there must be no state carried forward
between runs, just in case I have missed that during reviewing this entry.

**Comment by PanagiotisDrakatos** [comment]
@gunnarmorling I don't know why the first run takes so long but I can guarantee that every first run of all other people that i tried who make less than 5 sec takes too long 15-20 sec for example. Could you please check it if possible thanks in advance.

**Comment by PanagiotisDrakatos** [comment]
@gunnarmorling Basically i will change again the code no need to bother you again i will return back with a new solution :)
